modules/ztp-baremetal-host-requirements-factory.adoc

```diff
@@ -0,0 +1,122 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-du-host-requirements_{context}"]
+= Distributed unit host requirements
```

Comment:
Also I'm not seeing this module included anywhere. Should it be included, or removed?

Reply:
need to remove